debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnector.java

```diff
@@ -56,4 +67,54 @@ public void stop() {
     public ConfigDef config() {
         return SqlServerConnectorConfig.configDef();
     }
+
+    @Override
+    public Config validate(Map<String, String> connectorConfigs) {
+        Configuration config = Configuration.from(connectorConfigs);
+        SqlServerConnectorConfig sqlServerConfig = new SqlServerConnectorConfig(config);
+        // First, validate all of the individual fields, which is easy since don't make any of the fields invisible ...
+        Map<String, ConfigValue> results = config.validate(SqlServerConnectorConfig.ALL_FIELDS);
+
+        // Get the config values for each of the connection-related fields ...
+        ConfigValue hostnameValue = results.get(SqlServerConnectorConfig.HOSTNAME.name());
```

Comment:
Let's access the fields via defining classes, i.e. `RelationalDatabaseConnectorConfig.HOSTNAME` etc.